<!DOCTYPE html>
<html>
	<head>
		<title>David Díez, PhD.</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
  ga('create', 'UA-66632595-1', 'auto');
  ga('send', 'pageview');
</script></head>
	<body>
	<div id='header'>
		<nav>
    		<ul>
    			<li><a class='two' href="/">Home</a></li>
	        	<li><a class='two' href="/about">About me</a></li>
        		<li><a class='two' href="/cv">CV</a></li>
        		<li><a class='two' href="/publications">Publications</a></li>
        		<li><a class='two' href="/projects">Projects</a></li>
        	<!--	<li><a class='two' href="/research">Research</a></li> -->
        		<li><a class='two' href="https://medium.com/@indianadiez">Blog</a></li>
    		</font>
    		</ul>
		</nav>
		</div><!-- /.header -->
		<div class="container">
    		<div class="blurb">
    			<a name='top'></a>
        		<h1>Projects</h1>
        		
<h2>Palaeogenomics of the woolly mammoth: a model to study near-extinction population dynamics</h2>

<p></p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/16173156/a3aa6a00-3598-11e6-9ead-03791915e2aa.jpg'  HEIGHT=274/>
    <figcaption>Woolly mammoths. Charles R. Knight, 1912. </figcaption>
</figure>

<p>...more details <a class='one' href="#Explaining human population history using ancient DNA"> here.</a></p>

<h2>Explaining human population history using ancient DNA</h2>

<p>I am interested in explaining past demographic events that have sculpted the human populations as we know them today. I approach this problem from a <i>population continuity</i> point of view. That is, can we actually prove that two populations inhabiting the same region but separated hundreds or thousands of years are continuous in time? I am developing statistical methods using DNA sequencing data from modern populations together with sequences from ancient populations extracted from human remains in archaeological sites to shed light on these questions.</p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9109857/f3b388ae-3c30-11e5-9d6b-32cb8bb47f7f.png' WIDTH=332 	HEIGHT=187/>
    <figcaption>Proposed routes of migration by early farmers into Europe ~9,000−7000 years ago (Haak et al. 2015).</figcaption>
</figure>

<p>...more details <a class='one' href="#Explaining human population history using ancient DNA"> here.</a></p>


<h2>Genetics of biological invasions</h2>

<p> Biological invasions are one of the main threats to global biodiversity. I am interested in discovering the genetic basis of the evolutionary framework that underlies the successful life history of mosquitofish <i>(Gambusia holbrooki)</i>, as a model for studying the genetics of invasive species.</p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9036943/3f9133ca-39de-11e5-9b74-128adfb98419.jpg' WIDTH=282 	HEIGHT=158/>
   <figcaption>Female of <i>Gambusia holbrooki</i>.</figcaption>
</figure>

<p> ...more details <a class='one' href="#Genetics of biological invasions"> here.</a> </p>


<br/>
<br/>

<hr>

<br/>
<br/>

<a name = 'Explaining human population history using ancient DNA' ></a>
<h2>Explaining human population history using ancient DNA </h2>
<p>I am interested in explaining past demographic events that have sculpted human populations as we know them today. To do so we are using DNA sequencing data from modern populations together with sequences from ancient populations extracted from human remains in archaeological sites by using a combination of population genomics and data modelling via computer simulations.</p>

<p>As part of the BEAN (<i>Bridging the European and Anatolian Neolithic</i>) project I am interested in developing robust statistical methods to help explain one of the most complex events in human evolutionary history: the emergence of agriculture and sedentary life accompanied by an explosive increase in population size, a phenomenon known as the <i>Neolithic Transition.</i> As the project title suggests, we are analyzing modern and ancient Anatolian and European genomes to shed light on the Neolithic transition in Europe.</p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9109857/f3b388ae-3c30-11e5-9d6b-32cb8bb47f7f.png' WIDTH=664 	HEIGHT=374/>
    <figcaption>Proposed routes of migration by early farmers into Europe ~9,000−7000 years ago (Haak et al. 2015).</figcaption>
</figure>

<p>I approach this problem from a population continuity point of view. This is, can we actually prove that two populations inhabiting the same region but separated hundreds or thousands of years are continuous in time? Because it usually means different things in different disciplines the concept of <i>population continuity</i> can be a little shady. Because I work with genetic data I consider population continuity between two populations (one ancient and one modern) when, accounting only for the effects of genetic drift over time, we can simulate forward from the ancient population and recover the genetic differences that we observe between the real ancient and modern genomes. </i>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9292978/fbc904ee-440d-11e5-8f39-7d7a247e3a56.png' WIDTH=664 	HEIGHT=372/>
    <figcaption>Example of the importance of population size on genetic drift simulations. Lines represent allele trajectories of a single variant position starting at an allele frequency of 0.25. Black lines represent simulations for population size of 10,000, and grey lines for population size of 100. </figcaption>
</figure>

<p>If mere drift is not enough to explain these differences, then we should invoke other evolutionary processes to explain variation between the ancient and the modern population (selection or admixture). Selection can act over populations under a population continuity hypothesis but by using genomic data (thousands of SNPs, single nucleotide polymorphism) we are safe assuming that selection can't be responsible for the observed genome-wide differences. Therefore, if genetic drift alone can't explain the differences seen in the two test populations, there is no population continuity between them, and different levels of admixture with other populations (from some migrants to total replacement) played the most important role on the history of the populations in the region studied.</p>

<p>It is very interesting to apply this scheme to the Neolithic transition. Can genetic drift alone explain the differences between palaeolithic hunter-gatherers and neolithic farmers? Also, this population continuity methodology is very interesting to effectively test other relevant questions in human history. For example, continuity between Sardinians and European First Farmers, between ancient (Clovis, The Kenewick Man...) and modern Native Americans, Anglo-Saxons and modern British... and beyond humans (horses, wolves, boars...). </p>

<p><font size='2'><a class='one' href='#top'>Back to top</a></font></p>

<br/>
<br/>

<hr>

<br/>
<br/>

<a name='Genetics of biological invasions'></a>
<h2>Genetics of biological invasions</h2>
<p>Biological invasions are one of the main threats to global biodiversity. I am interested in discovering the genetic basis of the evolutionary framework that underlies the successful life history of mosquitofish <i>(Gambusia holbrooki)</i>, as a model of invasive species. <i>G. holbrooki</i> is a small freshwater fish native from the Southeastern coast of North America that was introduced worldwide because it was suggested to help control mosquito populations and hence to spread diseases such as malaria. To Europe, mosquitofish arrived in 1921 with the documented introduction of only 12 individuals to a pond in Spain. From there it was further introduced to other Mediterranean basins and more eastern countries.  </p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9036943/3f9133ca-39de-11e5-9b74-128adfb98419.jpg' WIDTH=564 	HEIGHT=316/>
   <figcaption>Female of <i>Gambusia holbrooki</i>.</figcaption>
</figure>

<p>Genetic data based on microsatellite loci and SNPs indicates that, despite bottlenecks associated to founder events, European mosquitofish populations harbour high levels of genetic diversity. These European populations are structured and display an isolation-by-distance pattern of differentiation. Also, alternative mtDNA haplotypes and differential allele frequencies suggested two European colonization routes: the original one registered on historical records and mentioned above, and a more recent one involving populations of Italy and Greece. </p>

<p><small><a class='one' href="http://link.springer.com/article/10.1007/s10530-009-9505-5" target="_blank"> Origin and genetic diversity of mosquitofish <i>(Gambusia holbrooki)</i> introduced to Europe (2010) Oriol Vidal, Emili García-Berthou, Pablo A. Tedesco and José-Luis García-Marín </a> </small></p>

<p><small><a class='one' href="http://onlinelibrary.wiley.com/doi/10.1111/j.1600-0633.2011.00527.x/abstract" target="_blank"> SNP diversity in introduced populations of the invasive <i>Gambusia holbrooki</i> (2012) Oriol Vidal, Nuria Sanz, Rosa-Maria Araguas, Raquel Fernández-Cebrian, David Diez-del-Molino and José-Luis García-Marín </a></small> </p>

<p><small><a class='one' href="http://link.springer.com/article/10.1007%2Fs10530-013-0456-5" target="_blank"> Genetic characterization of the invasive mosquitofish <i>(Gambusia spp.)</i> introduced to Europe: population structure and colonization routes (2013) N. Sanz, R. M. Araguas, O. Vidal, D. Diez-del-Molino, R. Fernández-Cebrián and J. L. García-Marín</a></small> </p>

<p> Finding high levels of genetic diversity in introduced populations of mosquitofish is surprising because the general theory predicts the opposite (with some noteworthy <a class='one' href='http://www.nature.com/nature/journal/v431/n7005/abs/nature02807.html' target="_blank"> exceptions</a>). In fact populations from Mediterranean Spanish streams had similar levels of genetic diversity than American populations only when compared with the populations suggested to be the sources of the European introduction. Also, these introduced populations to Spain do not appear to have undergone substantial losses of genetic diversity during the invasion process. Additionally, these American sources allocated at the northern range of the natural distribution of the species probably appeared as a consequence of postglacial colonization, and nowadays display less genetic diversity for both microsatellite loci and mtDNA haplotypes than southern populations. We have proposed that these populations acted as <i>invasive bridgeheads</i> for the European invasion of mosquitofish, acquiring the evolutionary changes necessary to become invasive in the postglacial colonization process -long before the European invasion -, and  hence spreading easily in new environments once introduced by humans. We therefore believe that this bridgehead hypothesis could be applied to other invasive species, especially those arising from unstable environments such as post-glacially colonized areas.</p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9037242/4a7f63bc-39e1-11e5-854e-71d6a0982b55.png' WIDTH=664 	HEIGHT=554/>
    <figcaption>Invasive bridgehead hypothesis for northern American populations of <i>G. holbrooki</i>.</figcaption>
</figure>

<p>Mediterranean populations of mosquitofish in Spain display high levels of within river gene flow, which contributes to maintain the levels of genetic diversity. Gene flow among rivers has also been detected in areas of marshland in which populations from different rivers get connected in seasonal flooding periods. Strikingly, our results highlight the existence of artificial gene flow both within and among rivers mediated by human translocations. This artificial gene flow can have great impact on the maintenance or promotion of genetic diversity, allowing the species to spread to areas in which it is not yet present, and outcompete endemic species. These activities need to be specificity prohibited, controlled and sanctioned. </p>

<p><small><a class='one' href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0082501" target="_blank">Gene Flow and Maintenance of Genetic Diversity in Invasive Mosquitofish <i>(Gambusia holbrooki) </i> (2013) David Díez-del-Molino, Gerard Carmona-Catot, Rosa-Maria Araguas, Oriol Vidal, Nuria Sanz, Emili García-Berthou and Jose-Luis García-Marín </a></small> </p>

<p> <i> Other ongoing projects and collaborations </i></p>

<big>
<ul>
  <li><big>Human-mediated perturbations can affect genetic diversity and population structure of invasive species, promoting or obstructing their spread. Analyzing adaptive and neutral variation in mosquitofish populations inhabiting a highly perturbed reservoir (Flix, Ebroi River, Spain), we are studying how human-mediated perturbations such as pollutants and dams affect mosquitofish invasiveness.</big></li>
  
  <li><big>Literature shows that mosquitofish quickly adapt in response to environmental factors such as the presence of predators, salinity and thermal gradients, and pollutants. However, the temporal components of genetic diversity and geographical structure of invasive mosquitofish populations are poorly known. Using a temporal sampling scheme we aim to determine temporal patterns of genetic variation and dispersal of the species. First results point to a relative temporal stability on genetic diversity and population structure, but highly variable patterns of gene flow in short periods of time (1~2 generations). </big></li>

  <li><big>Using genotyping-by-sequencing (RAD-seq) technologies on several populations of mosquitofish, we are studying how the invasion process sculpts the genome of the species by comparing native American and invasive European populations. Preliminary results point to a prominent role of genetic drift in shaping the genome of invaders during the invasive process.</big></li>

  <li><big>Rapid adaptation to new environments has been detected in native areas of distribution of guppy and closely related species. In collaboration with  Bryan Neff & Shawn Gardner (University of Western Ontario, London, Canada) we are studying the hypervariable MHC loci of both native and invasive mosquitofish individuals aiming to detect the role of these rapid evolutionary events on the invasiveness of the species.</big></li>
</ul>
</big>
<p><font size='2'><a class='one' href='#top'>Back to top</a></font></p>


    		</div><!-- /.blurb -->
		</div><!-- /.container -->
		<footer>
    		<ul>
        		<li><a class='one' href="mailto:diez.molino@gmail.com">email</a></li>
        		<!-- <li><a class='one' href="https://github.com/diezmolino">github.com/diezmolino</a></li> -->
			</ul>
		</footer>
	</body>
</html>
