<!DOCTYPE html>
<html>
	<head>
		<title>David Díez, PhD.</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
	</head>
	<body>
		<nav>
    		<ul>
        		<li><a href="/">Home</a></li>
	        	<li><a href="/about">About me</a></li>
        		<li><a href="/cv">CV</a></li>
        		<li><a href="/research">Research</a></li>
        		<li><a href="/blog">Blog</a></li>
    		</ul>
		</nav>
		<div class="container">
    		<div class="blurb">
    		
<h1> David Díez-del-Molino </h1>

<p><a href="#Research interests"> Research interests</a></p>
<p><a href="#Contact details"> Contact details</a></p>
<p><a href="#Skills"> Skills</a></p>

<br/>
<br/>

<hr>

<br/>
<br/>


<a name='Research interests'></a>
<h2>Research interests</h2>

<p>I am interested in the study of animal and human populations through a wide range of mechanisms including population genetics, evolutive biology and computational genomics. For my thesis I have been studying evolutionary processes related to invasive species and their success in novel environments and I have acquired a wide range of skills useful for analyzing population genetics of almost any organism. I am currently involved in discovering the evolutionary processes that have contributed to shape the genomes of invaders and how these populations evolve from native to invaded environments. Nowadays my research is moving towards using genomic tools to investigate the origin and history of human populations through looking up to their genomes and modelling ancient DNA.</p>


<a name='Contact details'></a>
<h2>Contact details</h2>

<big>
<nav>
<ul>
<li><a href='https://uk.linkedin.com/pub/david-díez-del-molino/2b/711/66
'>LinkedIN</a></li>
<li><a href='https://twitter.com/IndianaDiez'>Twitter</a></li>
<li><a href='https://www.facebook.com/Daviddiez'>Facebook</a></li>
</ul>
</nav>
</big>

<p>email:
<big>
<nav>
<ul>
<li><a href="mailto:diez.molino@gmail.com">diez.molino (at) gmail.com</a></li>
<li><a href="mailto:d.molino@ucl.ac.uk">d.molino (at) ucl.ac.uk</a></li>
</ul>
</nav>
</big>

<a name='Skills'></a>
<h2>Skills</h2>
<font size="5">
<ul>
<li>Languages: Spanish/English/Catalan</li>

<li>Wet-lab work: techniques related to molecular markers such as microsatellite loci, SNPs, mtDNA, and NGS.</li>

<li>Software (Plink, ADMIXTURE, TreeMix, etc…) and analyses (FST, He, IBD, Tajima’s D, PSMC, etc…) related with population genetics/genomics.</li>

<li>Genomic data analysis: from handling big sequence files to variant calling, processing and filtering.</li>

<li>Computer simulations and modelling of ancient DNA data using either custom scripting or standard software such as <i>ms</i> or <i>FastSimCoal2</i>.</li>

<li>R (scripting and plotting), Python (scripting) and the GNU/Linux environment (including shell and cluster scripting).</li>

<li>Experience supervising undergraduate and master students.</li>

<li>Experience in writing projects and technical reports, communicating results in conferences and publishing in journals.</li>
</font>
</ul>

<a name='Education and Training'></a>
<h2>Education and Training</h2>

<font size="5">
<ul>
<li>2014—present: Postdoctoral Research Associate. MACE-lab. University College London.</li>

<li>2010—2015: Ph.D. in Population Genetics. Laboratori d’Ictiologia Genètica. University of Girona.</li>

<li>2008—2009: Master in Biological Conservation. The Complutense University of Madrid.</li>

<li>2003—2008: Degree in Biological Sciences. University of Salamanca.</li>

</ul>
</font>


<a name='Other Training / Recent Courses'></a>
<h2>Other Training / Recent Courses</h2>

<font size="5">
<ul>
<li>2013 — Next-Generation Sequencing data analysis . COMAV/Polytechnic University of Valencia, Valencia, Spain. (38 hours)</li>

<li>2014 — Computing for Data Analysis (R). Johns Hopkins University/Coursera (MOOC-4 weeks)</li>

<li>2014 — Bioinformatics Workshop. EMBL-EBI. University of Girona, Spain (2 days)</li>

<li>2014 — Introduction to High Performance High Throughput Computing. ISD/University College London. London, UK. (2 days)</li>

<li>2015 — Introduction to Genomic data analysis using Hapmap and the 1000 Genomes projects. Transmitting Science. Sabadell, Spain (5 days)</li>

<li>2015 — Variant Analysis with GATK. The Broad Institute/University of Cambridge. Cambridge, UK (2 days)</li>

<li>2015 — Population Genetics and Demographic History: model-based approaches. Instituto Gulbenkian de Ciência. Oeiras, Portugal (4 days)</li>
</ul>
</font>

<a name='Teaching Experience'></a>
<h2>Teaching Experience</h2>

<p>2013/2014: Adjunct Professor. University of Girona.</p>

<font size="5">
<ul>
<li>Integrated Scientific Techniques (46.5 hours)</li>

<li>Conservation Genetics (7.5 hours)</li>

<li>Practice in Genetics (35 hours)</li>

<li>Population Genetics (7.5 hours)</li>
</ul>
</font>







<p><small><a href="http://link.springer.com/article/10.1007/s10530-009-9505-5" target="_blank"> Origin and genetic diversity of mosquitofish <i>(Gambusia holbrooki)</i> introduced to Europe (2010) Oriol Vidal, Emili García-Berthou, Pablo A. Tedesco and José-Luis García-Marín </a> </small></p>

<figure>
    <img src='https://cloud.githubusercontent.com/assets/13287919/9037242/4a7f63bc-39e1-11e5-854e-71d6a0982b55.png' WIDTH=664 	HEIGHT=554/>
    <figcaption>Invasive bridgehead hypothesis for northern American populations of <i>G. holbrooki</i>.</figcaption>
</figure>



    		</div><!-- /.blurb -->
		</div><!-- /.container -->
		<footer>
    		<ul>
        		<li><a href="mailto:diez.molino@gmail.com">email</a></li>
        		<li><a href="https://github.com/diezmolino">github.com/diezmolino</a></li>
			</ul>
		</footer>
	</body>
</html>
